### PR TITLE
[DE] Update `Disambiguation`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Proposal to bring the German article for `Disambiguation` to its latest version.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) (__I'm a native myself__)

### CODE REVIEW

**Comment by TicClick** [comment]
> I'm a native myself

nah, it has to be another person, that's what the review is for

**Comment by TicClick** [comment]
(same goes for other PRs of yours)

File: wiki/Disambiguation/de.md [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+- [Combo](Combo)
+- [Fail](Fail)
+- [NC](NC)
+- [NM](NM)
+- [Offset](Offset)
+- [osu!](osu!)
+- [Ouendan](Ouendan)
+- [Rank](Rank)
+- [Star](Star)
+- [Version](Version)
```
Comment (FamousPig) [inline wiki/Disambiguation/de.md:22]: The english list is sorted alphabetically, I believe this one should be the same.

**Review by FamousPig — CHANGES_REQUESTED** [review]
Everything else looks good to me 👍.

File: wiki/Disambiguation/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Disambiguation/de.md:22]: Yeah, seems plausible. I just copy-pasted the missing ones 🤷

**Review by FamousPig — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**